siriuspy/siriuspy/orbintlk/main.py

```diff
@@ -1461,7 +1456,9 @@ def _check_minsum_requirement(self, monit_sum=None):
         facq_sum = monit_sum * self._get_bpm_rates_factor()
         return _np.all(facq_sum > self._limits['minsum'])
 
-    def _handle_reliability_failure(self):
+    def _handle_reliability_failure(self, is_failure=True):
+        if is_failure:
```

Comment:
shouldn't this if come after the check for the state?

Reply:
Ok, but, is it a fatal error when the state is not on?

Reply:
Hi guys, I changed to warning when the state is not on